src/components/footer/footer.styles.js

```diff
@@ -0,0 +1,56 @@
+import { createUseStyles } from 'react-jss'
+
+export const useFooterStyles = createUseStyles(theme => ({
+  footer: {
+    background: theme.palette.black,
+    color: theme.palette.gray.lighter,
+    fontSize: theme.spacing(2)
+  },
+  content: {
+    width: '100%',
+    padding: `${theme.spacing(11)}px ${theme.spacing(6)}px ${theme.spacing(6)}px`
+  },
+  topSection: {
+    display: 'flex',
+    '& div:first-child': {
+      maxWidth: 'calc(100vw/5)', // TODO: update when Chami sends new design
+      marginRight: 'auto',
+      '& :first-child, & :last-child': {
+        color: theme.palette.white
+      },
+      '& :first-child': {
+        fontWeight: theme.fontWeights.bold,
+        fontSize: theme.spacing(3),
+        lineHeight: `${theme.spacing(4)}px`
+      },
+      '& > *': {
+        lineHeight: `${theme.spacing(3)}px`,
+        marginBottom: theme.spacing(2)
+      }
+    },
+    '& ul': {
```

Comment:
Can't the div, ul and li have class names?

Reply:
They can, but I was trying to avoid excess naming. If you guys prefer that all elements are referenced by a class, I'll add. Let me know @elias-garcia @AlbertoElias

Reply:
Yep, I also prefer to have dedicated classnames. I think that it's easier to follow when reading through the HTML

Reply:
Refactored. More refactors like these will follow in upcoming PRs.